detection-rules/link_credential_phishing_voicemail_language.yml

```diff
@@ -103,9 +103,17 @@ source: |
     "magicjack.com",
     "unitelvoice.com",
     "voipinterface.net",
-    "ringcentral.biz"
+    "ringcentral.biz",
+    "verizonwireless.com",
+    "t-mobile.com"
   )
   and not any(attachments, strings.starts_with(.content_type, "audio"))
+  // filter out topics that mention audio and listening, but are not voicemails
+  and not any(["podcast", "tune", "album", "episode", "study", "series", "coach"],
+              any([subject.subject, sender.display_name, body.current_thread.text],
```

Comment:
I like this idea, but this current implementation causes FNs when any of `subject.subject, sender.display_name, body.current_thread.text` are null (display_name and body.current_thread.txt both observed causing FNs)

It might be worth adding this logic to the existing check of these fields as seen/proposed in: https://github.com/sublime-security/sublime-rules/pull/1870/files.  This avoids looping the same details twice and allows for more specific inspection of the otherwise matching field.

Reply:
Just to be clear, is that something you plan on doing or would you like me to make the changes to your PR?

Reply:
@zoomequipd just updated, what do you think of this revised version?

Reply:
🎯  that's _exactly_ what I was thinking.